### PR TITLE
Integrate real runner execution usage

### PR DESCRIPTION
**Pull request by arbulu89**
Implement the real runner check execution request process.

It includes:
- New table with host connection settings. By now, it only contains the connection user. This table is empty always now, and it will be populated from the frontend, using some sort of form, as we do now in old trento
- Post execution request message to the runner
- Include the `skipped` state to the check results enum

TODO:
- Handle check results with `unreachable` state in the hosts
- Provide the correct provider for the execution
- Store the `user` from the web frontend

### CODE REVIEW

File: lib/trento/application/event_handlers/checks_event_handler.ex [hunk 1]
```diff
@@ -16,18 +17,23 @@ defmodule Trento.ChecksEventHandler do
         %ChecksExecutionRequested{cluster_id: cluster_id, hosts: hosts, checks: checks},
         %{correlation_id: execution_id}
       ) do
-    case Checks.request_execution(execution_id, cluster_id, hosts, checks) do
+
+    hosts_settings =
+      Enum.map(hosts, fn host -> Hosts.get_connection_settings(host) end)
+      |> Enum.filter(fn host -> host != nil end)
+
```
Comment (fabriziosestito) [inline lib/trento/application/event_handlers/checks_event_handler.ex:23]: `Enum.filter(& &1)`

File: lib/trento/application/event_handlers/checks_event_handler.ex [hunk 2]
```diff
@@ -16,18 +17,23 @@ defmodule Trento.ChecksEventHandler do
         %ChecksExecutionRequested{cluster_id: cluster_id, hosts: hosts, checks: checks},
         %{correlation_id: execution_id}
       ) do
-    case Checks.request_execution(execution_id, cluster_id, hosts, checks) do
+
+    hosts_settings =
+      Enum.map(hosts, fn host -> Hosts.get_connection_settings(host) end)
```
Comment (fabriziosestito) [inline lib/trento/application/event_handlers/checks_event_handler.ex:21]: it is  idiomatic to start the pipe with the variable e.g. `hosts |> Enum.map(fn host-> .....`

File: lib/trento/domain/cluster/events/checks_execution_requested.ex [hunk 3]
```diff
@@ -6,8 +6,8 @@ defmodule Trento.Domain.Events.ChecksExecutionRequested do
   use Trento.Event
 
   defevent do
-    field :cluster_id, :string
-    field :hosts, {:array, :string}
+    field :cluster_id, Ecto.UUID
+    field :hosts, {:array, Ecto.UUID}
```
Comment (fabriziosestito) [inline lib/trento/domain/cluster/events/checks_execution_requested.ex:10]: great! i missed this

**Review by nelsonkopliku — APPROVED** [review]
Looks good :rocket:

**Comment by arbulu89** [comment]
@fabriziosestito All the changes done! Thank you

**Pull request merged**